### PR TITLE
anydesk 6.1.1 update

### LINKED ISSUE #54: Update to version 6.1.1
New version was released. Please see related PR #53 and #55.

### PR DESCRIPTION
**Pull request by amz-x**
Required changes and dependencies for version 6.1.1

Fixes #54

### CODE REVIEW

**Comment by amz-x** [comment]
@wjt how do I get flathubbot to build the latest commit?

**Comment by amz-x** [comment]
@wjt I've added the missing dependencies, application now opens. Testable with the [build 45459](https://github.com/flathub/com.anydesk.Anydesk/pull/55#issuecomment-822682368).

**Comment by amz-x** [comment]
@wjt I've tested this build for the past two days. Looks good.

File: com.anydesk.Anydesk.json [hunk 1]
```diff
@@ -28,6 +28,25 @@
...
+            "name": "zlib",
+            "build-commands": [
+                "./configure --prefix=/app",
+                "make",
+                "make install",
+                "cd ./contrib/minizip && autoreconf --install",
+                "cd ./contrib/minizip && ./configure --prefix=/app",
+                "cd ./contrib/minizip && make",
+                "cd ./contrib/minizip && make install"
+            ],
```
Comment (wjt) [inline com.anydesk.Anydesk.json:41]: zlib is included  in the freedesktop runtime – indeed it's exactly the same version, 1.2.11. I think you can just build the `contrib/minizip` subdir.

See https://github.com/flathub/org.qgis.qgis/blob/d94ead7cbba1123e9a467a04391e866e27924e3a/org.qgis.qgis.json#L278-L295 https://github.com/flathub/org.wireshark.Wireshark/blob/b52d415985f08a7a0673ee3efb4e5f59a1193aeb/org.wireshark.Wireshark.yml#L22-L36 for examples.

**Review by wjt — CHANGES_REQUESTED** [review]
One minor quibble, otherwise looks good – thanks!

File: com.anydesk.Anydesk.json (hunk 1 above)
Comment (amz-x) [reply com.anydesk.Anydesk.json]: @wjt thank you for the examples. Updated the code as requested. Latest build doesn't seem to have issues.

**Review by wjt — APPROVED** [review]

**Comment by wjt** [comment]
Many thanks! Are you interesting in adopting this wrapper?

**Comment by amz-x** [comment]
@wjt I might be interested, however this was my first flatpak contribution.

**Pull request merged**